src/group/mls_group/api.rs

```diff
@@ -72,7 +72,7 @@ pub trait Api: Sized {
         &mut self,
         mls_plaintext: MLSPlaintext,
         proposals: Vec<MLSPlaintext>,
-        own_key_packages: Vec<KeyPackageBundle>,
+        own_key_packages: &Vec<KeyPackageBundle>,
```

Comment:
What clippy says. But we might actually need the vector. Please check. Same below.

Reply:
I changed it and everything compiles/tests/checks just fine. Anything else I need to look out for?